.pylintrc

```diff
@@ -74,6 +74,7 @@ disable=
     R0914, # Too many local variables (%s/%s)
     R0915, # Too many statements
     C0415, # Import outside toplevel (%s) Used when an import statement is used anywhere other than the module toplevel. Move this import to the top of the file.
+    C0115, # missing-class-docstring
```

Comment:
Why does this have to be disabled?

Reply:
I won't be able to add docstring to all the classes